docs/implementation_plans/2026-02-28_domain_logging_agent_runtime_fix.md

```
@@ -0,0 +1,268 @@
+  void log(String domain, String message, {String? subDomain, InsightLevel level});
+  void error(String domain, String message, {Object? error, StackTrace? stackTrace});
+  static String sanitizeId(String id); // → [id:abc123]
+  static String sanitizeContent(String content); // → [content: 142 chars]
+}
+```
+
+### 1b. Per-Domain Config Flags
+
+Add domain-specific logging flags to `lib/utils/consts.dart`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)